sdk/servicebus/azure-servicebus/azure/servicebus/control_client/servicebusservice.py

```diff
@@ -1111,7 +1111,8 @@ def create_event_hub(self, hub_name, hub=None, fail_on_exist=False):
         request = HTTPRequest()
         request.method = 'PUT'
         request.host = self._get_host()
-        request.path = '/' + _str(hub_name) + '?api-version=2014-01'
+        #request.path = '/' + _str(hub_name) + '?api-version=2014-01'
```

Comment:
Are these commented lines here for ongoing debugging?
Are we now using a default API version rather than specifying the query param?

Reply:
I agree with Anna, if we don't need these lines it's not incorrect push of debug mode, let's remove them explicitly